subtensor-api/src/subtensorapi/cli.py

```diff
@@ -0,0 +1,71 @@
+# The MIT License (MIT)
```

Comment:
Check the path of these files.

I think it should be `subtensor-api/subtensorapi/cli-py`, Is that right?